notebooks/Testgrid_flakiness_detection.ipynb

```diff
@@ -0,0 +1,2907 @@
+{
```

Comment:
<p>Can you make sure all the graphs are the same size?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
These are the test cases from testgrid repo. I do not want to change the test cases. 
Hence, I removed the graph of this section and just keep 'pass/fail' status.

Reply:
<p>I just meant the figure sizes should all be the same. But it looks like there are all at least the same width now. </p>